.deploy/nais/app-prod.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: familie-baks-infotrygd-feed
+  namespace: teamfamilie
+  labels:
+    team: teamfamilie
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-next-upstream-tries: '1'
+spec:
+  image: {{image}}
+  port: 8094
+  leaderElection: true
+  liveness:
+    path: /internal/health/liveness
+    initialDelay: 20
+    failureThreshold: 10
+  readiness:
+    path: /internal/health/readyness
+    initialDelay: 20
+    failureThreshold: 10
+  prometheus:
+    enabled: true
+    path: /internal/prometheus
+  vault:
+    enabled: false
+  gcp: # Database
+    sqlInstances:
+      - type: POSTGRES_14 # IF This is changed, all data will be lost. Read on nais.io how to upgrade
+        tier: db-custom-1-3840
+        name: familie-baks-infotrygd-feed
+        autoBackupTime: "02:00"
+        pointInTimeRecovery: true
+        diskAutoresize: true
+        highAvailability: true
+        databases:
+          - name: familie-baks-infotrygd-feed
+            envVarPrefix: DB
+  azure:
+    application:
+      enabled: true
+      allowAllUsers: false
```

Comment:
tror fortsatt denne må være satt til true for at infotrygd skal kunne kalle oss. også er det sjekk tilgang som sjekker om `access_as_application` rollen er satt. Men er jo ikke noe problem å teste med false først.